tests/resources/helper_functions.py

```diff
@@ -345,3 +346,37 @@ def transfer_ocean_if_balance_lte(
         )
 
     return ocean_token.balanceOf(recipient) - initial_recipient_balance
+
+
+def get_provider_fees() -> Dict[str, Any]:
```

Comment:
Shouldn't you get the provider fees from the provider itself?

Reply:
To clarify: when testing the initialize function separately, this is perfect. So leave it in the code.
But when actually doing the flow (initialize+download), please use the provider fees you get from the initialize step.